youtube_dl/extractor/pandoratv.py

```diff
@@ -0,0 +1,40 @@
+# encoding: utf-8
```

Comment:
There must be [`from __future__ import unicode_literals` on top](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site).